packages/module/src/common/constants.ts

```diff
@@ -1,6 +1,6 @@
-import { csl } from '@mesh/core';
-import type { Costmdls } from '@mesh/core';
-import type { Budget, Era, Network, Protocol } from './types';
+import { csl } from '../core/index.js';
```

Comment:
We probably could have achieved a basic level of Deno support (via CDNs which fix-up some ESM issues) just by no longer using the `@mesh/*` alias. The need for the `.js` suffixes is a result of setting `moduleResolution` to `Node16`, which additionally guarantees compatibility with Deno's built-in `npm` support, but which also ensures that support will continue to work in the future too.